apstools/utils/override_parameters.py

```diff
@@ -63,57 +66,46 @@ class OverrideParameters:
     Define parameters that can be overridden from a user configuration file.
 
     NOTE: This is a pure Python object, not using ophyd.
-
-    .. autosummary::
```

Comment:
These `autosummary` sections are used by Restructured Text and Sphinx to generate source code documentation.  Please leave them in.  #1081 will address.